### ISSUE TITLE
Documentation with API reference

### ISSUE THREAD

**Issue by erez-o** [jbeder/yaml-cpp#934]
Hi,

I've created a documentation with API reference,  you can see it at https://yaml-cpp.docsforge.com/

Hope it's helpful for anyone using yaml-cpp. The docs are built automatically every week so they should always be in sync with the code and wiki docs.

If there are any more tutorial pages you wish to add to the sidebar, or you wish to hide certain internal APIs from being displayed please let me know, everything is configurable using a simple yaml configuration file. I can transfer ownership and you can configure everything.

Disclaimer - I'm the creator of docsforge, free service for open source projects.

**Comment by RajaSrinivasan** [comment]
This doc is just a copy of what is in this repo right? Does it add any additional examples?

**Comment by erez-o** [comment]
Yes, it's only derived from what's inside the repo with api references.

**Comment by acwn1976** [comment]
Thank you for that. It's actually more helpful than the original documentation, although it has the same source. :+1:

**Comment by L0ric0** [comment]
this is far more usefull as it excludes the googletest namespace/classes
